### PR TITLE
Drop flatstr

### PR DESCRIPTION
**Pull request by mcollina**
So long, and thanks for all the fish `flatstr`.

(This is a semver-major PR)

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node-version: [10.x, 12.x, 13.x, 14.x, 15.x]
+        node-version: [12.x, 14.x]
```
Comment (kibertoad) [inline .github/workflows/nodejs.yml:11]: should we also add 16.x?

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
@mcollina Is there a reason why flatstr was not removed from package.json, btw?

**Comment by mcollina** [comment]
none

**Comment by mcollina** [comment]
can you update it?

**Comment by kibertoad** [comment]
Done: https://github.com/pinojs/sonic-boom/pull/71

**Pull request merged**